substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java

```diff
@@ -82,7 +82,7 @@ public boolean consume(Queue<String> args) {
                 if (!NativeImage.graalvmConfig.isEmpty()) {
                     message += " " + NativeImage.graalvmConfig;
                 }
-                message += " (Java Version " + System.getProperty("java.version") + ")";
```

Comment:
`java.version` and `java.vm.version` are 2 different pieces of info:
```
java -XshowSettings:properties  2>&1| grep 'java.*version'
    java.class.version = 52.0
    java.runtime.version = 1.8.0_251-b08
    java.specification.version = 1.8
    java.version = 1.8.0_251
    java.vm.specification.version = 1.8
    java.vm.version = 25.251-b08-jvmci-20.2-b01
```
People may already be depending on the current output format and semantics. This is API and evolving it should be done carefully.

Reply:
You are right. I only checked with java 11.
Changing this to a draft PR for the time being.
Thanks @dougxc